fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1041,6 +1043,14 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = GLOBAL_RUNTIME_FILTER_RPC_HTTP_MIN_SIZE, flag = VariableMgr.INVISIBLE)
     private long globalRuntimeFilterRpcHttpMinSize = 64L * 1024 * 1024;
 
+    /**
+     * The maximum number of rows of building hash table side to generate the runtime in filters.
+     * <p> If the number of rows of a HashJoinBuildOperator exceeds this threshold value, 
+     * the runtime in filters will not be generated.
+     */
+    @VariableMgr.VarAttr(name = RUNTIME_IN_FILTER_BUILD_MAX_SIZE, flag = VariableMgr.INVISIBLE)
+    private long runtimeInFilterBuildMaxSize = 128L * 1024L;
```

Comment:
maybe too large IN Filter?

Reply:
This is just for the limitation of the number of rows not distinct key values of the hash table.
The limitation of the distinct key values is still 1024.

Reply:
test